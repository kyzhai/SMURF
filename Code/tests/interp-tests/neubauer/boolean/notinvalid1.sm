a = 4
b = !a

main = []
