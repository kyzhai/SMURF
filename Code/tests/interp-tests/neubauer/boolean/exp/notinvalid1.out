Fatal error: exception Sast.Type_error("Element in Not operation must be of type Bool but element was of type Int")
